index.html

```diff
@@ -8788,7 +8788,7 @@
 	return A2(_elm_lang$core$List$filterMap, _jamonholmgren$rocket_elm$Bullet$tickBullet, bullets);
 };
 var _jamonholmgren$rocket_elm$Ship$shipView = function (ship) {
-	var rocketImg = (_elm_lang$core$Native_Utils.cmp(ship.acc, 0) > 0) ? './assets/rocket-3-burn.svg' : './assets/rocket-3.svg';
+	var rocketImg = (_elm_lang$core$Native_Utils.cmp(ship.acc, 0) > 0) ? './assets/Rocket-Revamped-Burn.png' : './assets/Rocket-Revamped.png';
```

Comment:
@denton85 This is the compiled file. Could you do the following from the project folder:

Edit `Ship.elm` to have the right filenames (also, rename to lower-case `rocket-4-burn.png` etc).

Then compile the file:

```
$ brew install elm
$ elm-make Main.elm
```